packages/pwa-buildpack/lib/__tests__/cli-create-project.spec.js

```diff
@@ -213,7 +213,107 @@ test('warns if backendurl does not match env', async () => {
     process.env.MAGENTO_BACKEND_URL = old;
 });
 
-test('runs install', async () => {
+test('if backendUrl matches env', async () => {
+    const old = process.env.MAGENTO_BACKEND_URL;
+    process.env.MAGENTO_BACKEND_URL = 'https://example.com';
+    fse.ensureDir.mockResolvedValueOnce(true);
+    fse.readdir.mockResolvedValueOnce(true);
+    await expect(
+        createProjectCliBuilder.handler({
+            backendUrl: 'https://example.com',
+            name: 'goo',
+            template: 'venia-concept',
+            directory: '/project',
+            npmClient: 'yarn'
+        })
+    ).resolves.not.toThrow();
+    expect(process.env.MAGENTO_BACKEND_URL).toBe('https://example.com');
+    process.env.MAGENTO_BACKEND_URL = old;
+});
+
+test('warns if backendEdition does not match env', async () => {
+    const old = process.env.MAGENTO_BACKEND_EDITION;
+    process.env.MAGENTO_BACKEND_EDITION = 'CE';
+    fse.ensureDir.mockResolvedValueOnce(true);
+    fse.readdir.mockResolvedValueOnce(true);
+    await expect(
+        createProjectCliBuilder.handler({
+            backendUrl: 'https://example.com',
+            backendEdition: 'EE',
+            name: 'goo',
+            template: 'venia-concept',
+            directory: '/project',
+            npmClient: 'yarn'
+        })
+    ).resolves.not.toThrow();
+    expect(process.env.MAGENTO_BACKEND_EDITION).not.toBe('EE');
+    expect(console.warn).toHaveBeenCalledWith(
+        expect.stringMatching('Environment variable overrides!')
+    );
+    process.env.MAGENTO_BACKEND_EDITION = old;
+});
+
+test('if backendEdition matches env', async () => {
+    const old = process.env.MAGENTO_BACKEND_EDITION;
+    process.env.MAGENTO_BACKEND_EDITION = 'EE';
+    fse.ensureDir.mockResolvedValueOnce(true);
+    fse.readdir.mockResolvedValueOnce(true);
+    await expect(
+        createProjectCliBuilder.handler({
+            backendUrl: 'EE',
```

Comment:
Should backendUrl be `https://example.com` instead?

Reply:
True, wrong copy paste. I'll fix that.

Reply:
@JustinConabreeAbsolunet Fixed.